rpcs3/Emu/Cell/PPUThread.cpp

```diff
@@ -914,6 +914,7 @@ void ppu_thread::exec_task()
 
 ppu_thread::~ppu_thread()
 {
+	perf_log.notice("Perf stats for STCX reload: successs %u, failure %u", last_succ, last_fail);
```

Comment:
@elad335, I suppose there is a typo in the message, should be `success` and not `successs`.